face.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+# Face.py for loading GIFs and images
+from PIL import Image
+
+# loads a gif into the array for storage
+
+
+def gif_loader(file):
+    arr = []
+    gif = Image.open(file, "r")
+    try:
+        while 1:
+            arr.append(gif.copy().convert("RGB"))
+            gif.seek(len(arr))
+    except EOFError:
+        return arr
+
+
+class Face:
+    def __init__(self):

```

Comment:
I'd recommend using `os.listdir()` to load gifs, and then reading their data into a dictionary. That way you don't have a million floating vars around, i.e.,
```
mouths = new dict()
for item in os.listdir("faces/mouth"):
  # Do something with item, where item is the filename, maybe parse out the .gif
  filename = item.split(".gif")[0]
  mouths[filename] = gif_loader("faces/mouth/" + item)
```
I would probably keep the mouths and eyes separate.